recipe/meta.yaml

```
@@ -112,7 +112,7 @@ outputs:
         - libarrow-dataset  # [libarrow != 13]
         - libparquet        # [libarrow != 13]
       run:
-        - libgdal-core >={{ ".".join(version.split(".")[:2]) }}
+        - libgdal-core =={{ ".".join(version.split(".")[:3]) }}
```

I believe the preferred way to to this would be:
```suggestion
        - {{ pin_subpackage('libgdal-core', exact=True) }}
```